guides/get-started-microservices-on-kubernetes/simple-microservice-part1.adoc

```diff
@@ -0,0 +1,252 @@
+= {simple-microservice-part1}
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+:prerequisites-time: 10
+
+In this guide, you will learn how to create and run a link:https://docs.docker.com/guides/docker-concepts/the-basics/what-is-an-image/[Docker Image] containing a link:https://jakarta.ee/specifications/restful-ws/[Jakarta RESTful Web Service] implemented using WildFly.
+
+include::{includedir}/_prerequisites.adoc[]
+
+* link:/get-started[Getting Started with WildFly]
+
```

Comment:
I'm missing "docker/podman" mentioned in prerequisites.

Reply:
@mnovak1 Tnak you, added